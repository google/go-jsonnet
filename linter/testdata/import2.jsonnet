(import '42.jsonnet')(17)
