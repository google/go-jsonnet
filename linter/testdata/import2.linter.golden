testdata/import2:1:1-26 Called value must be a function, but it is assumed to be a number

(import '42.jsonnet')(17)


